### PR TITLE
Alerts-Menu

### PR DESCRIPTION
**Pull request by RafifInph**

### CODE REVIEW

File: libs/app-shared/src/lib/components/alert-box-item.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  content: string,
+  date: Date;
+  read?: boolean;
+}
+
+export interface AlertBoxItemProps {
+    item: AlertBoxItemType,
+    onClickMenuItem: () => void,
+}
+
```
Comment (huanganya) [inline libs/app-shared/src/lib/components/alert-box-item.tsx:18]: //rafif todo:
//1, add data-testid to the title, date, content and action buttons
//2, add prop of swtich for the display the action buttons(announcement notification no need action buttons)
//3, add prop of redirect url to redirect the user to a new page, as some notification will make the user to some page with forms to submit

File: libs/app-shared/src/lib/components/alert-box-item.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  title: 'AppShared-AlertBoxItem',
+};
+export default Story;
+
+const Template: ComponentStory<typeof AlertBoxItem> = (args) => {
+  return <AlertBoxItem {...args} />;
+};
+
+export const Primary = Template.bind({});
+Primary.args = {
```
Comment (huanganya) [inline libs/app-shared/src/lib/components/alert-box-item.stories.tsx:15]: some other args are missing. The component cannot display in storybook

File: libs/app-shared/src/lib/components/alert-box-item.tsx (hunk 1 above)
Comment (RafifInph) [reply libs/app-shared/src/lib/components/alert-box-item.tsx:18]: - What the action button does? is it to redirect user to another page or is there any other function for that button?

Comment (RafifInph) [reply libs/app-shared/src/lib/components/alert-box-item.tsx:18]: does the action button used the prop from point , a prop of redirect url to redirect the user to a new page?

**Review by huanganya — APPROVED** [review]

**Pull request merged**